adding_species_manuscript.tex

```diff
@@ -209,23 +209,21 @@ \section*{Introduction}
 For instance, simulations provide training data
 for inference methods based on machine learning \citep{Schrider2018} and
 Approximate Bayesian Computation \citep{Csillery2010}. They can also serve as
-baselines for further analyses: for example, simulations incorporating
+baselines for further analyses: simulations incorporating
 demographic history serve as null models when detecting selection \citep{Hsieh2016a}
 or seed downstream breeding program simulations \citep{Gaynor2020}.
-More recently, population genomic simulations have begun
-to be used to help guide conservation decisions for threatened species
+More recently, population genomic simulations have been used to help guide conservation decisions for threatened species
 \citep{Teixeira2021,kyriazis2022using}.
 
 Increasing amounts of data and sophistication of inference methods
-have enabled researchers to ask ever more
```

Comment:
I prefer "ever," it implies that the specificity and precision will continue to increase in the future